app/src/docker/docker-compose.yml

```diff
@@ -198,6 +189,20 @@ services:
     networks:
       - intranet
 
+  svc-bip-api:
+    profiles: ["all", "svc", "bip"]
```

Comment:
@greene858 This `image` line is missing and causing `app:dockerComposeUp` to fail:
```
    image: va/abd_vro-svc-bip-api:latest
```